src/scss/objects/_links.scss

```diff
@@ -25,3 +25,10 @@ a {
     //     text-decoration: none;
     // }
 }
+
+.o-tile-link {
```

Comment:
This should likely be named something like `o-link--tile` or something similar.

Have you got an example of how it's being used?

Reply:
I've added changes as discussed